airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java

```diff
@@ -477,6 +483,24 @@ public int getNumberOfFileBuffers(final JsonNode config) {
   }
 
   public static void main(final String[] args) throws Exception {
+    if ("--write".equals(args[0])) {
+      final BigQueryException e = new BigQueryException(42, "edgao demo exception: oh no we couldn't find table foo.bar at [84:126]");
+      // This puts the mangled string into the trace external message and the original string into the internal message, which is backwards
+      // we'll need to add a new utility method to this class that does the right thing
+//      AirbyteTraceMessageUtility.emitSystemErrorTrace(e, "edgao demo exception: oh no we couldn't find table ?.? at [84:126]");
+      final Consumer<AirbyteMessage> outputRecordCollector = Destination::defaultOutputRecordCollector;
+      outputRecordCollector.accept(new AirbyteMessage()
+          .withType(AirbyteMessage.Type.TRACE)
+          .withTrace(new AirbyteTraceMessage()
+              .withEmittedAt((double) System.currentTimeMillis())
+              .withType(AirbyteTraceMessage.Type.ERROR)
+              .withError(new AirbyteErrorTraceMessage()
+                  .withMessage("edgao demo exception: oh no we couldn't find table foo.bar at [84:126]")
+                  .withInternalMessage("airbyte_destination_bigquery_modified: edgao demo exception: oh no we couldn't find table ?.? at [84:126]")
```

Comment:
this is what our sentry rule will match against. Note the placeholders for the dataset/table name, and the special prefix.